app/src/main/java/com/android/partagix/ui/screens/EditAccount.kt

```diff
@@ -157,20 +181,126 @@ fun EditAccount(
 
                 Spacer(modifier = Modifier.height(16.dp))
                 Row(
-                    modifier = Modifier.fillMaxWidth().padding(8.dp, 0.dp).testTag("actionButtons"),
+                    modifier =
+                        Modifier.padding(16.dp, 0.dp)
+                            .clickable { contact = !contact }
+                            .testTag("contactInfo"),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Absolute.Left) {
+                      if (contact) {
+                        Icon(imageVector = Icons.Default.KeyboardArrowUp, contentDescription = null)
+                      } else {
+                        Icon(
+                            imageVector = Icons.Default.KeyboardArrowDown,
+                            contentDescription = null)
+                      }
+                      Text("Contact information ")
+                    }
+
+                if (contact) {
+                  Spacer(modifier = Modifier.height(8.dp))
+
+                  TextField(
+                      modifier = Modifier.fillMaxWidth().padding(16.dp, 0.dp).testTag("email"),
+                      value = email,
```

Comment:
The entered information should fit on a single line. The same modification can be applied to the 2 other fileds.
```suggestion
                      value = email,
                      singleLine = true,
```

Reply:
yes you're right !